tests/README.md

```diff
@@ -39,23 +39,26 @@ func TestMyExampleComponent(t *testing.T) {
 	require.NotNil(t, expectedResourceMetrics)
 
 	// combination OTLP Receiver consumertests.MetricsSink
-	otlp, err := testutils.NewOTLPReceiverSink().WithEndpoint("localhost:23456").Build()
+	otlp, err := testutils.NewOTLPMetricsReceiverSink().WithEndpoint("localhost:23456").Build()
 	require.NoError(t, err)
 	require.NoError(t, otlp.Start())
 
 	defer func() {
-		require.Nil(t, otlp.Shutdown())
+		require.NoError(t, otlp.Shutdown())
 	}()
 
 	myContainer := testutils.NewContainer().WithImage("someTarget").Build()
 	err = myContainer.Start(context.Background())
 	require.NoError(t, err)
 
 	// running collector subprocess that uses the provided config set to export OTLP to our test receiver
-	myCollector, err := testutils.NewCollectorProcess().WithConfig(path.Join(".", "testdata", "config.yaml")).Build()
+	myCollector, err := testutils.NewCollectorProcess().WithConfigPath(path.Join(".", "testdata", "config.yaml")).Build()
```

Comment:
Do we use hard-coded port numbers in the provided config.yaml? If so, consider auto-generating config using available port numbers. See Collector testbed on the approach. Not for this PR, just suggesting as a future improvement.

Reply:
Yes, hardcoded is the only supported path atm, but I'll be sure to add config from string functionality to use the testbed available port helper and support test parameterization.